tests/Dummy/TestQueueManager.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Spatie\FailedJobMonitor\Test\Dummy;
+
+use Illuminate\Contracts\Queue\Job;
+use Illuminate\Queue\Jobs\SyncJob;
+
+class TestQueueManager
+{
+    protected $container;
+
+    public function __construct($container)
+    {
+        $this->container = $container;
+    }
+
+    protected function createPayload($job, $data = '', $queue = null) : string
+    {
+        if (is_object($job)) {
+            $payload = json_encode([
+                'job' => 'Illuminate\Queue\CallQueuedHandler@call',
+                'data' => [
+                    'commandName' => get_class($job),
+                    'command' => serialize(clone $job),
+                ],
+            ]);
```

Comment:
I really dislike the use of `else` in general. Could you early return the `if` branch?